Mathlib/Analysis/Convex/ConvexFunction.lean

```diff
@@ -0,0 +1,541 @@
+/-
+Copyright (c) 2023 Chenyi Li, Ziyu Wang, Zaiwen Wen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chenyi Li, Ziyu Wang, Penghao Yu, Zhipeng Cao, Shengyang Xu, Zaiwen Wen
+-/
+import Mathlib.Analysis.Calculus.FDeriv.Basic
+import Mathlib.Analysis.InnerProductSpace.Dual
+import Mathlib.Analysis.Convex.Function
+import Mathlib.Analysis.Calculus.MeanValue
+import Mathlib.Analysis.Calculus.Gradient.Basic
+
+/-!
+## Main results
+
+This file contains the following parts of smooth convex functions.
+* (First Order Condition for convex functions)
+    Function f(x) is convex if and only if for any x, y ∈ s, f(y) ≥ f(x) + f'(x) (y - x)
+* (Monotonity of Gradient)
+    Function f(x) is convex if and only if its gradient is monotone, i.e.
+    for any x, y ∈ s, (f'(x) - f'(y)) (x - y) ≥ 0
+* (Corresponding Result for strict convex functions)
+    Function f(x) is convex if and only if its gradient is strictly monotone, i.e.
+    for any x, y ∈ s, x ≠ y, (f'(x) - f'(y)) (x - y) ≥ 0
```

Comment:
Did you mean `>` here for "strictly"? I would also suggest using backticks so that your formulas will render as code on the docs page. Or you can use LaTeX with dollar signs.